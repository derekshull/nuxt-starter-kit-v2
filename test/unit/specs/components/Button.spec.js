import { shallowMount } from "@vue/test-utils";
import Component from "~/components/Button.vue";

describe("Component Button", () => {
  it("renders to a snapshot", () => {
    const wrapper = shallowMount(Component);
    expect(wrapper.element).toMatchSnapshot();
  });
});
