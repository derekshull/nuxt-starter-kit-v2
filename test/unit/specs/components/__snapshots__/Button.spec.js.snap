// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Button renders to a snapshot 1`] = `
<button
  class="button"
  style="color: rgb(66, 185, 131); border-color: #42b983;"
>
  !
</button>
`;
